orchestrator/ethereum_gravity/src/logic_call.rs

```diff
@@ -52,7 +54,7 @@ pub async fn send_eth_logic_call(
             before_nonce
         );
         return Ok(());
-    } else if current_block_height > call.timeout.into() {
+    } else if current_block_height > Uint256::from_u64(call.timeout) {
```

Comment:
any specific reason why we don't provide `into()` method for Uint256?

Reply:
IIRC I did it because `uint` did not, and in Rust you cannot implement foreign traits for foreign types. In most cases `into` for integer casting does not save many characters because you need to specify `: Uint256` and add a `uX` suffix to the literal so Rust can infer both sides. `from_uX` is also more explicit.

Reply:
yes I preferer being explicit but just noting that nothing is stopping you from implementing From<u64> and Into<u64> for your own Uint256 type